src/libpriv/rpmostree-passwd-util.c

```diff
@@ -409,6 +410,8 @@ rpmostree_check_passwd_groups (gboolean         passwd,
             goto out;
           
           old_path = g_file_resolve_relative_path (root, commit_filepath);
+          if (!g_file_load_contents (old_path, cancellable, &old_contents, NULL, NULL, error))
+            goto out;
```

Comment:
Ouch, I see, dangerous :zap: stuff. That :bug: would have passed right by if you weren't testing in Docker!
Could you add a comment :speech_balloon: maybe to explain the use of `g_file_load_contents()`?